### PR TITLE
Add a hook to allow pre transformers hook

### PR DESCRIPTION
**Pull request by ChrisPRobinson**
[#150620006]

### CODE REVIEW

File: lib/etl/output/redshift2.rb [hunk 1]
```diff
@@ -22,29 +22,53 @@ def initialize(client, main_table_name, optional_history_table_name, surrogate_k
       @id_generator = id_generator || IDUUIDGenerator.new
       @now_generator = ::ETL::Output::CurrentDateTimeGenerator.new
       raise "main_table_name must not be empty" if @main_table_name.empty?
+    end
+
+    def build_transformer(table_schemas_lookup)
+      transformers = []
+      transformers << pre_transformer if !@pre_transformer.nil?
```
Comment (emya) [inline lib/etl/output/redshift2.rb]: `@pre_transformer`

Comment (emya) [inline lib/etl/output/redshift2.rb]: need to put '@' to `pre_transformer` ?

Comment (ChrisPRobinson) [reply lib/etl/output/redshift2.rb]: Yes, going to add a test to make sure it all works.

Comment (ChrisPRobinson) [reply lib/etl/output/redshift2.rb]: Fixed

**Pull request merged**